interfaces/HO-Portal/src/app/components/registration-activity-overview/registration-activity-overview.component.html

```diff
@@ -0,0 +1,86 @@
+<ion-text>
+  <h1 class="ion-padding-start">
+    {{ 'registration-details.activity-overview.title' | translate }}
+  </h1>
+</ion-text>
+
+<ion-card>
+  <ion-item
+    color="light"
+    lines="none"
+  >
+    <ion-row style="width: 100%">
+      <ion-col
+        [sizeXs]="12"
+        [sizeLg]="9"
+        class="horizontal-scroll-container"
+      >
+        <div class="ion-text-nowrap">
+          <ion-button
+            *ngFor="let filter of activityOverviewButtons"
+            (click)="activityOverviewFilter = filter"
+            fill="clear"
+            size="small"
+            [color]="
+              activityOverviewFilter === filter ? 'primary' : 'secondary'
+            "
+            [style]="
+              activityOverviewFilter === filter
+                ? 'box-shadow: 0px 2px var(--ion-color-primary); --ripple-color: #fff'
+                : '--ripple-color: #fff'
+            "
+          >
+            {{
+              ('registration-details.activity-overview.filterButton.' + filter
+                | translate) +
+                ' (' +
+                getFilterCount(filter) +
+                ')'
+            }}
+          </ion-button>
+        </div>
+      </ion-col>
+    </ion-row>
+  </ion-item>
+  <ion-card-content class="ion-no-padding">
+    <ion-list
+      class="ion-no-padding"
+      lines="full"
+    >
+      <ion-item
+        *ngFor="let item of getFilteredActivityOverview()"
+        style="font-size: 14px; --padding-start: 0"
+      >
+        <ion-grid>
+          <ion-row>
+            <div class="ion-padding-horizontal">
+              <div style="height: 100%; display: flex; align-items: center">
+                <ion-icon
+                  [name]="getIconName(item.type)"
```

Comment:
This will break the production-build in that it won't detect the icons-in-use, so their files will not be included on the server...

We can work around that, I think...